src/project/internal/notationproject.cpp

```diff
@@ -592,10 +593,21 @@ Ret NotationProject::doSave(const muse::io::path_t& path, engraving::MscIoMode i
         IF_ASSERT_FAILED(params.mode != MscIoMode::Unknown) {
             return make_ret(Ret::Code::InternalError);
         }
+        if (ioMode == MscIoMode::Zip
+            && globalConfiguration()->devModeEnabled()
+            && savePath.contains(" - CORRUPTED.mscz")
```

Comment:
Perhaps we should use a more sophisticated suffix, for example `- ALL_ZEROS_CORRUPTED.mscz`. To reduce even a slight chance of accidentally corrupting the score